language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -80,7 +78,7 @@ newtype Script = Script {unScript :: T.Text}
 data Options = Options
     { optInputDars :: [FilePath]
     , optOutputDir :: FilePath
-    , optInputPackageJson :: FilePath
+    , optInputPackageJson :: Maybe FilePath -- Deprecated.
```

Comment:
Can you please create an issue to make sure we remove this flag after the next release.

Reply:
https://github.com/digital-asset/daml/issues/5194